### PR TITLE
Create flat model

### PR DESCRIPTION
**Pull request by ctln12**
Generate flat model with name and address
Run migration
Add presence validations

### CODE REVIEW

File: db/migrate/20201122135514_create_flats.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateFlats < ActiveRecord::Migration[6.0]
+  def change
+    create_table :flats do |t|
+      t.string :name
+      t.string :address
+
```
Comment (ctln12) [inline db/migrate/20201122135514_create_flats.rb:6]: It would be maybe nice to have a city field too ?

**Review by ctln12 — COMMENTED** [review]
Could you have a look at my comment and make changes if necessary ?

File: app/models/flat.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class Flat < ApplicationRecord
```
Comment (batch280) [inline app/models/flat.rb:1]: Could you add some validations please ?

**Review by batch280 — CHANGES_REQUESTED** [review]
Have a look at my comment and make the necessary changes. Thanks!

File: app/models/flat.rb (hunk 2 above)
Comment (ctln12) [reply app/models/flat.rb:1]: Yes sure

**Pull request merged**